packages/components/src/core/styles/common/mixins/a11y.ts

```diff
@@ -0,0 +1,8 @@
+export function focusVisibleA11yStyle() {
```

Comment:
should we make this util function depend on `props`, so `5px` is not hard coded and can use `spaces`?

Reply:
unfortunately spaces does not have a 5px option